templates/borg-backup.timer.epp

```diff
@@ -1,12 +1,17 @@
+<%- | Hash[String,String] $backuptime,
+| -%>
 # THIS FILE IS MANAGED BY PUPPET
 [Unit]
-Description=Daily Timer for borg-backup
+Description=Daily Timer(s) for borg-backup
 
 [Timer]
 # OnBootSec=10min
 # OnUnitActiveSec=1d
-# Daily at 8am
-OnCalendar=*-*-* 18:30:00
+<% include stdlib -%>
+<% $backuptime.keys.each | $key | { -%>
+# Daily named job "<%= $key %>" at <%= $backuptime[$key] %>
+OnCalendar=*-*-* <%= $backuptime[$key] %>
```

Comment:
can you please add a unit test that verifies the file content is correct?

Reply:
I have added a couple of unit tests to verify the syntax and processing (also processing of multiple backuptimes), but I am not experienced with rspec-puppet to be able to actually verify the correct OnCalendar output.
Would you be able to help me a bit with this (or direct me to the documentation on how I can verify the File in a dependant function call * )?

* What I mean is, the camptocamp/systemd is a dependancy of this module, which is not a problem, but I don't know how to verify the content of the EPP with this.